enn.py

```diff
@@ -33,41 +36,112 @@ def __init__(self, *args, **kwargs):
 
 
 
+    def inference_nas(self, train_loader, valid_loader):
+        for idx, value in enumerate(self.model.replaced_modules):
+            for number_of_rsacm in range(1,self.args.number_of_rsacm+1):
+                with torch.no_grad():
+                    get_module_by_name(self.model, value).override_forward(number_of_rsacm)
```

Comment:
I am not sure that I understand this correctly function - this line sets the number of used rsacms only for the current moddule ("value" variable, from the outer loop)? So we choose the number of rsacms from 1 to 8 sequentially? Shouldn't we reset the number of rsacms (to one or max) for all other modules (other than the current one) in each iteration?

Reply:
By default the ACCM uses all the RSACMs (8). This codes goes sequentially ACCM by ACCM. After it goes to another ACCM you can say that it reset the number of rsacms because the previous ACCM has it rsacm set to 8.